Gems/Multiplayer/Code/Include/Multiplayer/IMultiplayerConnectionViewportMessage.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+
+namespace Multiplayer
+{
+    //! Interface class for displaying useful Multiplayer debug information to the center of the viewport.
+    //! Example: it can show status as the editor is making its connection to the editor-server.
+    class IMultiplayerConnectionViewportMessage
+    {
+    public:        
+        AZ_RTTI(IMultiplayerConnectionViewportMessage, "{83ca783b-4e99-46c9-a133-7d1e4bacdbb4}");
+
+        //! Displays multiplayer debug messaging in the viewport.
+        //! This will appear in the center of the screen.
+        virtual void DisplayCenterViewportMessage(const char* text) = 0;
```

Comment:
What about providing an offset here as an option? The center of the screen is too specific for a generic interface here, imho.

Reply:
Good call! Making me think... Setting this text will override previous text, so allowing users to define the location would give the false sense that this text is independent. Maybe this class has given the wrong impression; it's not meant to be an interface for ppl to use. I should really close it from the outside world (similar to how I dont allow users to define the bottom-right connection status and instead the class generates that itself). The center text is only meant to show the editor->server steps which didn't have an ebus I could listen into so I exposed this API directly.

Reply:
Created a bug to fix this; it'll take a bit of more code so will post this in a new PR. 
https://github.com/o3de/o3de/issues/10614